server/test/helper.js

```diff
@@ -3,12 +3,23 @@
 
 import helper from 'fastify-cli/helper.js';
 import path from 'path';
+import * as nodemailerMock from 'nodemailer-mock';
+import quibble from 'quibble';
 import { fileURLToPath } from 'url';
 
+import { PrismaClient } from '@prisma/client';
+
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 const AppPath = path.join(__dirname, '..', 'app.js');
 
+// Environment variable overrides/additions for testing
+process.env.NODE_ENV = 'test';
```

Comment:
By setting NODE_ENV to test, we have a way for code to check if they're running in test or not. Should be used sparingly, only as a last resort when there's no other way to handle test environment differences (i.e. like with mocking, see below).